examples/kv-server/src/main/java/com/yahoo/gondola/demo/DemoService.java

```diff
@@ -25,20 +32,19 @@
 
     public DemoService() throws Exception {
         // Create the Gondola instance
+
         URL gondolaConfURI = DemoApplication.class.getClassLoader().getResource("gondola.conf");
```

Comment:
Both URI/Uri can be used, such as javax.servlet - HttpServletRequest, it uses URI in capital.

https://tomcat.apache.org/tomcat-5.5-doc/servletapi/javax/servlet/http/HttpServletRequest.html#getRequestURI()